src/data/levels.js

```diff
@@ -29,6 +29,10 @@ levels[6] = {
   unlocksTool: toolType.SHOVEL,
 }
 
+levels[7] = {
+  unlocksShopItem: items.garlicSeed.id,
+}
```

Comment:
What do you think about keeping the level 7 reward slot open for now and moving this up to something like level 34? I think having gaps in the static level rewards brings a few advantages:

  - It makes for a more unique experience for each player earlier in the game, as [levels that don't have a predefined reward just give the player a random seed](https://github.com/jeremyckahn/farmhand/blob/5c87bb6e604cb00c9ed2dc289ec606e5fa862b1c/src/utils.js#L887-L892).
  - It adds to the late game experience which (hopefully) keeps players coming back more over the long term.

Reply:
ahh yes i totally forgot about the random seed reward. i'm fine with changing it to a level 34 reward!